manifests/base/resources/precedents/secrets/selectivepatch.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: qlik.com/v1
```

Comment:
remove resources folder altogether, as all the secrets is referenced from .operator folder